doc/lua_api.md

```diff
@@ -7772,6 +7772,10 @@ child will follow movement and rotation of that bone.
     * `stat` supports the same keys as in the hud definition table except for
       `"hud_elem_type"`.
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
+* `hud_get_elements()`:
```

Comment:
Since `hud_get` is not called `hud_get_element`, `hud_remove` is not called `hud_remove_element` etc., it is inconsistent naming that the name of `hud_get_elements` contains the word "elements".

`hud_get_all` would be more consistent.

Reply:
The HUD does not only consist of elements so I thought `hud_get_elements` would be more descriptive, but for consistency `hud_get_all` is definitely better.

It is changed now.